### ISSUE TITLE
E2504 enforces incorrect boundary 100-2000, should be 100-20000

### ISSUE THREAD

**Issue by chloehutchinson** [awslabs/cfn-python-lint#688]
*cfn-lint version: (`cfn-lint --version`)* 0.15.0 (also tested with 0.14.1)

**Description:**
AWS EC2 Instance with block device mapping does not enforce the correct `Ebs/Iops` boundary of 100-20000 as specified in the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-blockdev-template.html)

It looks like it actually enforces 100-2000, as if you set `Iops` to 2000 it passes the linter.

**Sample to reproduce:**
```yaml
Resources:
  Machine:
    Type: AWS::EC2::Instance
    Properties:
      ImageId: "ami-79fd7eee"
      KeyName: "testkey"
      BlockDeviceMappings: 
        - DeviceName: /dev/sdm
          Ebs: 
            VolumeType: io1
            Iops: 3000 
            DeleteOnTermination: false
            VolumeSize: 20
```

Output:
```bash
> cfn-lint minimal-stack.yaml

E2504 Property Iops should be Int between 100 to 20000 Resources/Machine/Properties/BlockDeviceMappings/0/Ebs/Iops
minimal-stack.yaml:11:13
```

**Comment by chloehutchinson** [comment]
See line 53 for the issue https://github.com/awslabs/cfn-python-lint/blob/f5c3504f593d4311388a047fa4b6f18f792c919a/src/cfnlint/rules/resources/ectwo/Ebs.py#L50-L55
